csm_web/frontend/src/components/App.tsx

```diff
@@ -28,39 +29,38 @@ interface ErrorPageProps {
   clearError: () => void;
 }
 
-export default class App extends React.Component {
-  state: AppState = { error: null };
+export default function App() {
+  const state: AppState = { error: null };
 
-  static getDerivedStateFromError(error: ErrorType): { error: ErrorType } {
+  const getDerivedStateFromError = (error: ErrorType): { error: ErrorType } => {
     return { error };
-  }
-
```

Comment:
These changes for the `App.tsx` should be reverted (aside from the addition of the policies page); this needs to be a class component in order to implement error boundaries (it's why it's still a class component now).

Reply:
reverted back to class. good practice with git revert 🙏